funnel/models/commentvote.py

```diff
@@ -80,12 +82,14 @@ class Commentset(BaseMixin, db.Model):
     type = db.Column(db.Integer, nullable=True)
     count = db.Column(db.Integer, default=0, nullable=False)
 
+    proposal = db.relationship("Proposal", back_populates="commentset", uselist=False)
```

Comment:
Now I'm thinking this might not be right. Was there a reason for making a single commentset/voteset being able to relate with multiple proposals?

Reply:
Commentsets can be linked to multiple models, not just proposals.